AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -35,7 +41,10 @@ let package = Package(
         .watchOS(.v6)
     ],
     products: [
-        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"])
+        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"]),
+        .library(name: "AWSSDKEventStreamsAuth", targets: ["AWSSDKEventStreamsAuth"]),
```

Comment:
Good with either `AWS` prefix or `SDK` prefix. My preference is actually `SDK` because it is clearly. I.e. this is **our** runtime / our code not something determined by AWS service teams

Reply:
I used `AWSSDK` as the prefix for SDK runtime modules because all our service client modules are prefixed with only `AWS`.

For example, I can easily see an AWS service being launched with the name `Identity`... you can see the potential for namespace collisions.

So using `AWSSDK` as the prefix for runtime names helps to prevent namespace collisions between SDK components and AWS service clients.